main.js

```diff
@@ -93,115 +90,166 @@
+            iconAnchor: [16, 32]
+        })
+    };
+
+    const WPRDC_DATA_SOURCES = {
+        "Police": {
+            id: '1797ead8-8262-41cc-9099-cbc8a161924b',
+            primaryFiltering: 'WHERE "INCIDENTNEIGHBORHOOD" LIKE \'%Oakland\'',
+            latLong: ['Y', 'X'],
+            icon: iconTypes.CITY_POLICE,
+
+            popup: (record) => record['OFFENSES'],
+
+            processRecord: (record) => {
+                // Collect time of incident from the record
+                record.incidentYear = parseInt(record.INCIDENTTIME.substring(0,4));
+                record.incidentMonth = parseInt(record.INCIDENTTIME.substring(5,8));
+                record.incidentDay = parseInt(record.INCIDENTTIME.substring(8,10));
+            }
+        },
+
+        // City of Pittsburgh 311 data
+        // TODO: would be great to prune 311 data to the last 30 days, like the police data
+        "311": {
+            id: '40776043-ad00-40f5-9dc8-1fde865ff571',
+            primaryFiltering: 'WHERE "NEIGHBORHOOD" LIKE \'%Oakland\' ORDER BY "CREATED_ON" DESC',
+            latLong: ['Y', 'X'],
+            icon: iconTypes.CITY_311_ICON,
+
+            popup: (record) => record['REQUEST_TYPE'],
+
+            processRecord: (record) => {
+                // Collect time of incident from the record
+                record.incidentYear = parseInt(record.CREATED_ON.substring(0,4));
+                record.incidentMonth = parseInt(record.CREATED_ON.substring(5,8));
+                record.incidentDay = parseInt(record.CREATED_ON.substring(8,10));
+            }
+        },
+
+        // Calls from the library db
+        "Library": {
+          id: "2ba0788a-2f35-43aa-a47c-89c75f55cf9d",
+          primaryFiltering: 'WHERE "Name" LIKE \'%OAKLAND%\'',
+          latLong: ['Lat', 'Lon'],
+          icon: iconTypes.LIBRARY_ICON,
+
+          title: (record) => record['Name'],
+          popup: (record) => `
+            <strong>${record.Name}</strong>
+            <br> Address: ${record.Address}
+            <br> Phone: ${record.Phone}
+            <br> Monday: ${record.MoOpen.substring(0, 5)} - ${record.MoClose.substring(0, 5)}
+            <br> Tuesday: ${record.TuOpen.substring(0, 5)} - ${record.TuClose.substring(0, 5)}
+            <br> Wednesday: ${record.WeOpen.substring(0, 5)} - ${record.WeClose.substring(0, 5)}
+            <br> Thursday: ${record.ThOpen.substring(0, 5)} - ${record.ThClose.substring(0, 5)}
+            <br> Friday: ${record.FrOpen.substring(0, 5)} - ${record.FrClose.substring(0, 5)}
+            <br> Saturday: ${record.SaOpen.substring(0, 5)} - ${record.SaClose.substring(0, 5)}
+            <br> Sunday: ${record.SuOpen.substring(0, 5)} - ${record.SuClose.substring(0, 5)}
+            `
+        }
+    };
+
+    const WPRDC_QUERY_PREFIX = 'SELECT * FROM "';
+    const WPRDC_QUERY_SUFFIX = '" ';
+
+    // Fetch data from West Pennsylvania Regional Data Center using the SQL API
+    function fetchWPRDCData(dataSourceName, options) {
```

Comment:
FWIW, we can set default JS params in ES6, like:

 `function fetchWPRDCData(dataSourceName, options={}) {`

So we don't have to have the conditional after. We're being selective in what es6 features we use, so this is fine as is. Just FYI.